Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -854,108 +884,27 @@ PhysicalParticleContainer::AddPlasma (int lev, RealBox part_realbox)
                     // the next generated particle.
 
                     // include ballistic correction for plasma species with bulk motion
-                    const Real z0 = applyBallisticCorrection(pos, inj_mom, gamma_boost,
+                    amrex::Real z0 = applyBallisticCorrection(pos, inj_mom, gamma_boost,
```

Comment:
is z0 not const anymore?

Reply:
no! thishas to be `const`!! I forgot to revert it after doing some local tests where I had tried to "move" particles within the spherical ring. 
Thanks for catching this!!!!